jobs/gorouter/templates/pre-start.erb

```diff
@@ -90,3 +90,7 @@ ulimit -n <%= p("router.max_open_files") %>
 # Add jq to path
 cp /var/vcap/jobs/gorouter/bin/setup-jq /etc/profile.d/jq.sh
 chown root:vcap /etc/profile.d/jq.sh
+
+<% if spec.bootstrap == true %>
```

Comment:
Do we want to do this for ONLY the canary?  Is there any downside to making it run on every instance?  What happens if they don't use a canary?

Reply:
Bosh should always have 1 instance that is bootstrap instance and in case of canary it will be the bootstrap one. If there is no canary then it will just run it on one of them.

Reply:
Awesome, thanks for the clarification!